src/ImageDeleteModal.jsx

```diff
@@ -55,7 +56,7 @@ export const ImageDeleteModal = ({ imageWillDelete, onAddNotification }) => {
         if (all)
             client.delImage(imageWillDelete.isSystem, imageWillDelete.Id, false)
                     .catch(ex => {
-                        Dialogs.show(<ForceRemoveModal name={imageWillDelete.RepoTags[0]}
+                        Dialogs.show(<ForceRemoveModal name={repoTags[0] ?? _("intermediate image")}
```

Comment:
Here we should already know about `isIntermediateImage` so I would prefer using that.